src/benchmarklib/tpcds/tpcds_table_generator.cpp

```diff
@@ -312,56 +310,56 @@ std::unordered_map<std::string, BenchmarkTableInfo> TpcdsTableGenerator::generat
   return table_info_by_name;
 }
 
-std::shared_ptr<Table> TpcdsTableGenerator::_generate_table(const std::string& table_name) {
+std::shared_ptr<Table> TpcdsTableGenerator::generate_table(const std::string& table_name, ds_key_t max_rows) const {
   if (table_name == "call_center") {
-    return generate_call_center();
+    return generate_call_center(max_rows);
```

Comment:
I might be missing something, but where is this used in this PR?

Reply:
It's not used, I don't know how these changes got in here. I cleaned that up.